test/ComparedQueryable.Test/ComparedQueryableTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace ComparedQueryable.Test
+{
+    public class ComparedQueryableTest

```

Comment:
Can we have another test where the numbers that distinguish between the values are not at the beginning of the strings? It might be "Option 2" vs "Option 10".